src/encoded/static/components/viz/QuickInfoBar.js

```diff
@@ -190,64 +155,94 @@ export default class QuickInfoBar extends React.PureComponent {
-                    <div className="graph-icon" onMouseEnter={null /*_.debounce(()=>{ this.setState({ show : 'mosaicCharts' }); },1000)*/}>
-                        <i className="icon icon-pie-chart fas" style={{ opacity : 0.05 }} />
-                    </div>
-                </div>
+    render(){
+        const { id, isLoadingChartData, browseBaseState, href, context, className } = this.props;
+        const { show, mounted } = this.state;
+        const anyFiltersSet = this.memoized.anyFiltersSet(context, browseBaseState);
+        const invisible = this.memoized.isInvisibleForHref(href);
+        const browseBaseParams = navigate.getBrowseBaseParams(browseBaseState);
+        const expSetFilters = this.memoized.expSetFilters((context && context.filters) || null, browseBaseParams);
+
+        let outerClassName = "explanation";
+        if (typeof className === 'string') outerClassName += ' ' + className;
+        if (invisible) outerClassName += ' invisible';
+
+        let innerClassName = "inner container";
+        if (show !== false) innerClassName += ' showing';
+        if (show === 'activeFilters') innerClassName += ' showing-filters';
+
+        let innerBody;
+        if (!mounted || invisible) {
+            innerBody = (
+                <QuickInfoBarBody {...this.props} {...this.state} {...{ anyFiltersSet, expSetFilters }} />
             );
         } else {
-            return null;
+            innerBody = (
+                <ChartDataController.Provider id="quick_info_bar1">
+                    <QuickInfoBarBody {...this.props} {...this.state} {...{ anyFiltersSet, expSetFilters }} onIconMouseEnter={this.onIconMouseEnter}
+                        onBrowseStateToggle={this.onBrowseStateToggle} handleActiveFilterTermClick={this.handleActiveFilterTermClick} />
+                </ChartDataController.Provider>
+            );
         }
+
+        return (
+            <div id={id} className={outerClassName} key="outer">
+                <div className={innerClassName} onMouseLeave={this.onPanelAreaMouseLeave} key="inner">
+                    { innerBody }
+                </div>
+            </div>
+        );
     }
+}
 
-    render(){
-        const { id, isLoadingChartData, browseBaseState } = this.props;
-        const { show, mounted, togglingBrowseState } = this.state;
-        const anyFiltersSet = this.anyFiltersSet();
-        if (!mounted) return null;
 
-        let className = "inner container";
-        if (show !== false) className += ' showing';
-        if (show === 'activeFilters') className += ' showing-filters';
-        if (show === 'mosaicCharts') className += ' showing-charts';
+function QuickInfoBarBody (props) {
+    const {
+        context, schemas, expSetFilters,
+        isLoadingChartData, browseBaseState, togglingBrowseState,
+        show, reallyShow, mounted, anyFiltersSet,
+        onIconMouseEnter, onBrowseStateToggle, handleActiveFilterTermClick
```

Comment:
Codacy found an issue: ['onIconMouseEnter' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6178047)